src/main/java/frc/robot/RobotContainer.java

```diff
@@ -116,6 +128,12 @@ private void setupDriveTrain() {
     }
 
     public void putShuffleboardCommands() {
+        SmartShuffleboard.putCommand("Ramp", "SetArmPID400", new RampMove(ramp, 400));
```

Comment:
Why does your branch have all these SmartShuffleboard commands? Please do the following:

* Remove any commands that you did not create
* Create a new "DEPLOYER_DEBUG" constant, and wrap the two "Deployer" commands in an if (Constants.DEPLOYER_DEBUG) check